packages/pkl-ts-parser/scripts/update-ts.mjs

```diff
@@ -0,0 +1,36 @@
+import { readFileSync, writeFileSync } from 'node:fs';
+
+main();
+
+function main() {
+  const path = 'src/antlr4/ts/PklTsParser.ts';
```

Comment:
is it not better to use a relative path?
`path.resolve(__dirname, ...)`

Reply:
This file is called from 'update-ts.sh`, which checks the current path at the start of the script.